prow/pluginhelp/hook/hook.go

```diff
@@ -58,22 +58,24 @@ type githubClient interface {
 	GetRepos(org string, isUser bool) ([]github.Repo, error)
 }
 
-type HelpAgent struct {
+// HelpHandler is a handler that generates and serve plugin help information.
+type HelpHandler struct {
```

Comment:
Why the rename?

Reply:
Not strictly required, HelpAgent only purpose was to serve an HTTP endpoint. Thought the name change made sense when writing the comment.

I agree that this change is noise in this PR, though.